dmgroups.tex

```diff
@@ -128,8 +128,8 @@ \subsection{DM Change Control Board \label{sect:dmccb}}
 	\end{itemize}
 \item Membership
 	\begin{itemize}
-	\item Chaired by the Systems Engineer
-	\item Members include the DM Software Architect (\S\ref{role:softarc}), DM Operations Architect (\S\ref{role:opsarc}), DM System Interfaces Scientist (\S\ref{role:dmis}), DM SQuaRE T/CAM (\S\ref{role:tcam}), DM Release Manager (\S\ref{role:dmrm}) and DM Project Manager (\S\ref{role:dmpm})
+	\item Chaired by the Systems Engineer (\secref{role:sysengineer}).
+	\item The DMCCB has the same membership as the Systems Engineering Team (\secref{sect:sysengt}).
```

Comment:
I think that may have been an oversight .. even in the original.   I would like  the same membership of these two groups to make it simple. People can tune out topics not relevant to them. So if we add DMSS we are pretty good. There is more wording to be done as we phase out interface scientist potentially (or the role diminishes) so another reason to have DMSS in.